lib/nerves_motd.ex

```diff
@@ -131,9 +131,12 @@ defmodule NervesMOTD do
     [_total, _used, _free, _shared, _buff, _available] = runtime_mod().memory_usage()
   end
 
-  @spec load_average() :: String.t()
+  @spec load_average() :: iolist()
   defp load_average() do
-    runtime_mod().load_average()
+    case runtime_mod().load_average() do
+      [a, b, c] -> :io_lib.format("~s ~s ~s", [a, b, c])
+      _ -> nil
```

Comment:
```suggestion
      _ -> []
```
`nil` will cause an exception since it's not an `iolist`. Maybe we should return `"error"` rather than nothing?

Reply:
OK, I'll simply do the pattern match so that wrong patterns will raise.